shell/app/common/components/erda-icon/erda-icon.tsx

```diff
@@ -80,10 +80,21 @@ export const ErdaIcon = ({ className = '', onClick, iconType, ...rest }: IProps)
   return IconComp ? <IconComp className={className} onClick={onClick} {...rest} /> : <span>Not Exists</span>;
 };
 
-
-type CustomColor = 'primary' | 'light-primary' | 'shallow-primary' | 'white';
+type CustomColor =
+  | 'primary'
+  | 'light-primary'
+  | 'shallow-primary'
+  | 'white'
+  | 'shallow-gray'
+  | 'black-gray'
+  | 'shallow-gray-1'
+  | 'shallo-gray-2'
+  | 'danger-red'
+  | 'shallow-gray-3'
+  | 'pink';
```

Comment:
type CustomColor = keyof typeof COLOR is better

Reply:
Do we really need so many color ?

Reply:
iconpark property fill need